<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openfile_btn">
       <property name="text">
        <string>打开图片文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openvideo_btn">
       <property name="text">
        <string>打开视频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="opencap_btn">
       <property name="text">
        <string>打开摄像头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="process_btn">
       <property name="text">
        <string>图像处理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savefile_btn">
       <property name="text">
        <string>保存结果</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="showlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>521</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>显示窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
